### DISCUSSION TITLE
How to make a title become right-alignment ?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5414] [Help] [answered]
How to make a title table become right-alignment ?

I already make the row value become right-align, but don't how to make a title become right-align.

![image](https://user-images.githubusercontent.com/43429317/211180806-5175f186-5126-47b7-b432-326a5141dc3a.png)

thank you.

**Comment by danharrin** [comment, marked as answer]
`->alignRight()` should handle the heading and the cell content
